public/scripts/app.js

```diff
@@ -1,7 +1,83 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
+  console.log('app.js is loaded!');
+  $.ajax({
```

Comment:
Looks good here, the process to grab data from the database goes from 
1.  app.js, look for an ajax call
2.  look into server.js for a route that matches with method and url
3.  look at the route for a controller and function
4.  look into the controller for the function and add the logic to grab the data
5.  if you want to return JSON, you can do res.json(dataReturned) in your controller function or use success like you did here